test/ExpLibTest.sol

```diff
@@ -63,7 +62,7 @@ contract ExpLibTest is Test {
     }
 
     function testWExpWMulDownMaxRate() public pure {
-        ExpLib.wExp(ExpLib.WEXP_UPPER_BOUND).wMulDown(ConstantsLib.MAX_RATE_AT_TARGET);
```

Comment:
I do think the constants library has value here and in any repo on top using this repo as a submodule